int32 class_id
string class_name
float64 confidence
float64 x_min
float64 y_min
float64 x_max
float64 y_max